less/admin/DashboardPage.less

```diff
@@ -34,6 +34,10 @@
       &.item-tools {
         float: right;
         margin-right: 0;
+
+        .Button {
+          background: @body-bg;
+        }
```

Comment:
I think this would be best added inside `.Widget` to make it more generic, not having to do the same thing again in case we or an extension adds a button inside a widget.

Also might be better to use `.Button--color(@control-color, @body-bg)` instead